src/components/Login.html

```diff
@@ -1,5 +1,87 @@
 <!DOCTYPE html>
 <html lang="en">
+	<head>
+		<link
+			rel="stylesheet"
+			href="../styles/root.css"
+		/>
+	</head>
+	<body>
+		<div
+			id="myModal"
+			class="inputDomMainContainer flex-column-center"
+		>
+			<form
+				class="formContainerMain inputDomContainer AlertInfo input-flex inputError box-shadow"
+				action="submit"
```

Comment:
try to use code formatter like prettier for better indentation

Reply:
This indentation gave me by Prettier only